contracts/base/LoansBase.sol

```diff
@@ -568,18 +568,31 @@ contract LoansBase is LoansInterface, Base {
         internal
         view
         returns (uint256)
+    {
+        uint256 loanAmount = _getTotalOwed(loanID);
+        uint256 collateralRatio = loans[loanID].loanTerms.collateralRatio;
+        return loanAmount.mul(collateralRatio).div(TEN_THOUSAND);
+    }
+
+    /**
+        @notice Returns the total amount owed for a specified loan
+        @param loanID The id of the loan to get the total amount owed
+     */
+     function _getTotalOwed(uint256 loanID)
+        internal
+        view
+        returns (uint256)
     {
         TellerCommon.LoanStatus currentStatus = loans[loanID].status;
         uint256 loanAmount;
         if (currentStatus == TellerCommon.LoanStatus.TermsSet) {
-            loanAmount = loans[loanID].loanTerms.maxLoanAmount;
+            loanAmount = loans[loanID].loanTerms.maxLoanAmount.add(loans[loanID].interestOwed);
```

Comment:
Interest owed won't be set yet if the loan is not yet active. So we'll have to calculate the interest based on the max loan amount. 

Take the calculation code for the interest that is in `takeOutLoan` here:
https://github.com/teller-protocol/teller-protocol-v1/blob/72a6d1d17f77a1226a5f3a1855ae1fec2e02da04/contracts/base/LoansBase.sol#L254-L258
and maybe put it in its own internal function to reuse.

Reply:
Changed accordingly